lib/omnibus/packagers/msi.rb

```diff
@@ -73,9 +76,13 @@ class Packager::MSI < Packager::Base
             -cultures:en-us
             -loc "#{windows_safe_path(staging_dir, 'localization-en-us.wxl')}"
             project-files.wixobj source.wixobj
-            -out "#{windows_safe_path(Config.package_dir, package_name)}"
+            -out "#{msi_file}"
         EOH
         shellout!(light_command, returns: [0, 204])
+
+        if cert_name
```

Comment:
How about a compromise between the two:
```ruby
signing_identity 'My Signing Cert'
```
with an optional `store` parameter. `signtool.exe` defaults this to `My`.  So if your cert was in some other store:
```ruby
signing_identity "My Signing Cert", store: "FooStore"
```
Unless there is a particular reason to use the fingerprint